Library/Homebrew/utils/github.rb

```diff
@@ -508,29 +508,61 @@ def self.get_repo_license(user, repo)
     nil
   end
 
-  def self.fetch_pull_requests(name, tap_remote_repo, state: nil, version: nil)
+  def self.fetch_pull_requests(name, tap_remote_repo, state: nil, version: nil, merged: nil, created: nil)
     if version.present?
-      query = "#{name} #{version} is:pr"
+      query = "#{name} #{version}"
       regex = /(^|\s)#{Regexp.quote(name)}(:|,|\s)(.*\s)?#{Regexp.quote(version)}(:|,|\s|$)/i
     else
-      query = "#{name} is:pr"
+      query = name.to_s
       regex = /(^|\s)#{Regexp.quote(name)}(:|,|\s|$)/i
     end
-    issues_for_formula(query, tap_remote_repo: tap_remote_repo, state: state).select do |pr|
-      pr["html_url"].include?("/pull/") && regex.match?(pr["title"])
+    query += " is:pr"
+    query += " created:#{created}" if created
+    issues_for_formula(query, tap_remote_repo: tap_remote_repo, state: state).select do |issue|
```

Comment:
Does this use the search API? I assume not but just wanted to check because it's rate-limited so aggressively.

Reply:
Yes it does, same as the open/closed PR checks, but it's only called for throttled casks/formulae.

Reply:
I don't think it's a great idea to rely on this. As mentioned: the rate limits are already very low and stretched thin.